eclass/kernel-install.eclass

```diff
@@ -743,7 +728,7 @@ kernel-install_pkg_prerm() {
 kernel-install_pkg_postrm() {
 	debug-print-function ${FUNCNAME} "${@}"
 
-	if [[ -z ${ROOT} && ! ${KERNEL_IUSE_GENERIC_UKI} ]] && use initramfs; then
+	if [[ -z ${ROOT} && ! ${KERNEL_IUSE_GENERIC_UKI} ]]; then
```

Comment:
I'm not exactly happy about retroactive change of behavior but I care about `USE=-initramfs` so little that *whatever*.